src/oss-tests/ProjectManagerTests/NuGetProjectManagerTests.cs

```diff
@@ -23,37 +27,92 @@ public class NuGetProjectManagerTests
             { "https://api.nuget.org/v3/registration5-gz-semver2/razorengine/index.json", Resources.razorengine_json },
             { "https://api.nuget.org/v3/catalog0/data/2022.03.11.23.17.27/razorengine.4.2.3-beta1.json", Resources.razorengine_4_2_3_beta1_json },
         };
-
-        private readonly NuGetProjectManager _projectManager;
         
+        // Map PackageURLs to metadata as json.
+        private readonly IDictionary<string, string> _metadata = new Dictionary<string, string>()
+        {
+            { "pkg:nuget/razorengine@4.2.3-beta1", Resources.razorengine_4_2_3_beta1_metadata_json },
+            { "pkg:nuget/razorengine", Resources.razorengine_latest_metadata_json },
+        };
+        
+        // Map PackageURLs to the list of versions as json.
+        private readonly IDictionary<string, string> _versions = new Dictionary<string, string>()
+        {
+            { "pkg:nuget/razorengine@4.2.3-beta1", Resources.razorengine_versions_json },
+            { "pkg:nuget/razorengine", Resources.razorengine_versions_json },
+        };
```

Comment:
Nit: could these be immutable?
```suggestion
        // Map PackageURLs to metadata as json.
        private readonly IDictionary<string, string> _metadata = new Dictionary<string, string>()
        {
            { "pkg:nuget/razorengine@4.2.3-beta1", Resources.razorengine_4_2_3_beta1_metadata_json },
            { "pkg:nuget/razorengine", Resources.razorengine_latest_metadata_json },
        }.ToImmutableDictionary();
        
        // Map PackageURLs to the list of versions as json.
        private readonly IDictionary<string, string> _versions = new Dictionary<string, string>()
        {
            { "pkg:nuget/razorengine@4.2.3-beta1", Resources.razorengine_versions_json },
            { "pkg:nuget/razorengine", Resources.razorengine_versions_json },
        }.ToImmutableDictionary();
```